.devcontainer/devcontainer.json

```diff
@@ -8,6 +8,21 @@
 		"../compose.yml",
 		"compose.yml"
 	],
+	"features": {
+		"ghcr.io/devcontainers/features/common-utils:2": {
```

Comment:
Do we need this?

Reply:
Zsh make the terminal look nice. And also show the active branch so I think it is useful.

Reply:
Let's try it out. Zsh is close enough to Bash that I don't see any doc reason to not merge it. Worse case, we remove it later if it doesn't work out.